docs/README.md

```diff
@@ -179,3 +187,145 @@ Then run:
 ```bash
 make linkcheck
 ```
+
+## Live Code Blocks
+
+Live blocks enable readers to interact with and change Rego snippets from within the docs.
+They are written as standard markdown code blocks with a special syntax in the language label
+and are enabled used a postprocessing step during full site builds.
+
+> For them to render correctly you must build the site using `make serve` or Netlify.
+  If the live-docs build starts failing, try running `make live-docs-clear-cache`.
+  For the page to be able to contact the Rego Playground, you may also want to
+  disable your browser's CORS checks when developing, on Chrome this can be
+  accomplished using the `--disable-web-security` (along with `--user-data-dir`) flag.
```

Comment:
Is it possible for us to make the CORs stuff work with the netlify previews? As-is we need to do this for reviewing changes too which is less great.

Reply:
As it is they'll load in with the right output and hydrate correctly, but if you edit things you get a `can't contact server` error message in the output blocks (e.g. see [my fork on netlify](https://dboles-opa-docs.netlify.com/docs/edge/how-do-i-write-policies/)). I figured that was probably fine for both development and reviews (hence the "may want to" in the docs). If not, I'm not sure what to do... From a quick poke around, netlify doesn't seem to allow DNS settings for preview builds and it generates a new subdomains for them (e.g. `deploy-preview-1--dboles-opa-docs` instead of `deploy-preview-1.dboles-opa-docs`) so a regexp check in the playground to allow cors wouldn't work (e.g. someone could just name their netlify site `deploy-preview-42--dboles-opa-docs`).